nntrainer/tensor/blas_neon.cpp

```diff
@@ -701,6 +702,188 @@ void sgemv_neon_fp16(const __fp16 *A, const __fp16 *X, __fp16 *Y, uint32_t rows,
 void sgemv_transpose_neon_fp16(const __fp16 *A, const __fp16 *X, __fp16 *Y,
                                uint32_t rows, uint32_t cols, float alpha,
                                float beta) {
+  if (cols % 8 == 0 && cols >= OMP_THRESHOLD)
```

Comment:
How about setting the omp_num_threads = 1 for cols % 8 != 0 case? Then we don't need to make ```sgemv_transpose_neon_fp16_multithread```

Reply:
As far as I am concerned, we still need different function like `sgemv_transpose_neon_fp16_multithread` for best-optimization with microsecond scope since using multithreading for small GEMM/GEMV deteriorates latency.
Considering we probably can have both big GEMM/GEMV and small GEMM/GEMV in a single program at the same time, suggested change might behave differently with what we expect.